Owncloud iOs Client/Login/Login/UniversalLoginViewController.swift

```diff
@@ -845,73 +878,81 @@ public enum TextfieldType: String {
     
 // MARK: 'private' methods
     
-    func detectUserDataAndValidate(credentials: OCCredentialsDto, serverPath: String) {
-        if loginMode == .migrate {
+    func detectUserDataAndValidate(serverPath: String) {
```

Comment:
detectUserData**And**Validate if a method has an _and_  usually means that it could be splitted. In this case i think that one could be detectUserData, and then another one for validate whatever this validates.